### ISSUE TITLE
Use  the pretrained model Resnet34 cannot get the mentioned result?

### ISSUE THREAD

**Issue by iie-gao** [Cadene/pretrained-models.pytorch#111]
I used the pretrained model resnet34 (download from  https://download.pytorch.org/models/resnet34-333f7ec4.pth ) .   Using imagenet_eval.py testing on Imagenet only get Acc@1 72.342 Acc@5 90.844 not the mentioned 73.554  91.456 .  Why?

**Comment by Cadene** [comment]
On python 3.7.1, torch 0.4.1.post2 and pretrainedmodels 0.7.4 I get  * Acc@1 73.314 Acc@5 91.420.

I will soon add automated tests for this and upload my version of the validation set.
